sbncode/CAFMaker/FillReco.cxx

```diff
@@ -950,6 +970,59 @@ namespace caf
     srpfp.cnnscore.nclusters = cnnscore->nClusters;
   }
 
+  void FillPPFNuGraph(const std::vector<unsigned int> &sliceHitsMap,
+		      const std::vector<art::Ptr<anab::FeatureVector<1>>> &ngFilterResult,
+		      const std::vector<art::Ptr<anab::FeatureVector<5>>> &ngSemanticResult,
+		      const std::vector<art::Ptr<recob::Hit>> &pfpHits,
+		      caf::SRPFP& srpfp,
+		      bool allowEmpty)
+  {
+
+    auto arg_max = [](std::vector<float> const& vec) {
+      return std::distance(vec.begin(), std::max_element(vec.begin(), vec.end()));
+    };
```

Comment:
Another question on the SBN coding conventions: are lambdas okay within other methods? (If @cerati himself isn't sure then maybe @PetrilloAtWork is the best person to answer this (especially as he is also tagged as the other reviewer))

Reply:
There is nothing about this case in our guidelines.
Personally I do recommend the use of lambdas in methods where it makes sense (typically, to avoid the repetition of simple code). However, if the same lambda appears in different methods, then it should likely be a member function if possible.